MdePkg/Library/BaseLib/UnitTestHostBaseLib.inf

```diff
@@ -23,7 +23,7 @@
   LIBRARY_CLASS                  = UnitTestHostBaseLib|HOST_APPLICATION
 
 #
-#  VALID_ARCHITECTURES           = IA32 X64 EBC ARM AARCH64 RISCV64
+#  VALID_ARCHITECTURES           = IA32 X64

```

Comment:
This file includes sources for architectures being deleted here. Were they added by mistake?

Reply:
Yes.  They were added by mistake.  Components of type HOST_APPLICATION are only supported as Windows or Linux applications for IA32 and X64.  Additional host architecture application CPU types could be supported, but those would have to be added to the UnitTestFramrworkPkg and BaseTools to support those additional host CPU architectures.  If there is interest, then please enter a BZ feature request.